lib/team_statistics.rb

```diff
@@ -20,9 +20,8 @@ def best_season(team_id)
       end
     end
 
-    games_by_season = filtered_games.group_by do |game|
-      game.season
-    end
+    # MEMOIZATION
+    games_by_season = filtered_games.group_by(&:season)
```

Comment:
Neat!

Reply:
We can reuse this refactor in many other areas of our code during refactor time tomorrow.